ui/de-lib/src/main/java/org/iplantc/de/theme/base/client/admin/ontologies/OntologyDisplayStrings.properties

```diff
@@ -23,4 +25,10 @@ categorizeApp = Select tags for {0}
 appAvusCleared = Tags for {0} removed
 appClassified = {0} tagged with {1}
 appClassifiedList = {0} tagged with {1,list}
-emptyDEOntologyLabel = <NONE>
\ No newline at end of file
+emptyDEOntologyLabel = <NONE>
+hierarchyDeleted = {0} successfully removed
+confirmDeleteOntology = Are you sure you want to delete ontology version {0}?
+ontologyDeleted = Deleted {0} successfully
+rootColumnLabel = Iri(s)
+hierarchyColumnLabel = Root(s)
+confirmDeleteHierarchy = Are you sure you want to delete the {0} hierarchy?
```

Comment:
Would it be worth having a message with this confirmation letting the admin know that they can re-add the deleted sub-classes by re-saving the affected `root` hierarchy?
Although I'm not exactly sure how to word that more clearly.